### PR TITLE
T10707: Add $wgCleanSignatures to Meowpedia

### PR DESCRIPTION
**Pull request by brandon-wm**

### CODE REVIEW

**Comment by AgentIsai** [comment]
Why isn't this added to LocalSettings.php and ManageWikiSettings.php?

**Comment by thecutefinder** [comment]
@AgentIsai #5188

**Comment by brandon-wm** [comment]
@AgentIsai Wasn’t aware it was wanted to be a global option. If so can change.

File: LocalWiki.php [hunk 1]
```diff
@@ -419,5 +419,9 @@ function onBeforePageDisplay( OutputPage $out ) {
 	case 'worldboxwiki':
 		$wgSpecialPages['Analytics'] = DisabledSpecialPage::getCallback( 'Analytics', 'MatomoAnalytics-disabled' );
 
+		break;
+	case 'meowpediawiki':
+		$wgCleanSignatures = false;
```
Comment (redbluegreenhat) [inline LocalWiki.php:424]: This should go on LocalSettings.php instead imo. You can also just add it to ManageWiki if you want, but that's up to you.

Comment (brandon-wm) [reply LocalWiki.php:424]: Wouldn’t adding to LS.php set false as default for all wikis?

Comment (AgentIsai) [reply LocalWiki.php:424]: You can set the default value to true and then set the set it to false for that wiki only but in this case, it makes more sense to just add it into ManageWiki

Comment (brandon-wm) [reply LocalWiki.php:424]: As long as the default remains true, I have no objections. ManageWiki can be done.

Comment (redbluegreenhat) [reply LocalWiki.php:424]: Just noting that even if you're going the ManageWiki route, the default still needs to be set at LocalSettings, else it doesn't work properly.

**Comment by brandon-wm** [comment]
So.....moving to LS.php and implementing in ManageWiki?

**Comment by redbluegreenhat** [comment]
yes

**Pull request closed without merging**